config/default.yml

```
@@ -91,6 +91,11 @@ Lint/UselessAccessModifier:
     - concern
     - concerning
 
+Lint/UselessMethodDefinition:
+  Exclude:
```

Can you add a rationale for the `Exclude`? For example:

```suggestion
Lint/UselessMethodDefinition:
  # Avoids conflict with `Rails/LexicallyScopedActionFilter` cop.
  Exclude:
```